src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -7,6 +7,8 @@ Release History
 * (Breaking change): remove `assign-identity` & `remove-identity` which were tagged `deprecating` 2 releases ago
 * `vm create`: support configure Public-IP sku
 
+* vm resize will check if the requested size is different than currently set and update only on change
```

Comment:
Since we are past the code snap date for 2.0.30, please put it to a new section of `2.0.31` and update setup.py with this newer version.